themes/digital.gov/layouts/partials/core/home/news_featured.html

```diff
@@ -11,105 +11,94 @@ <h2>News and Events</h2>
             <p>Innovative work, news, and ideas from people and teams in government</p>
 
             <div class="join-buttons">
-              <a href="{{- "/join" | absURL -}}">Write for us</a>
+              <a href="{{- "/join" | relURL -}}">Write for us</a>
             </div>
 
           </header>
         </div>
       </div>
 
-      <div class="grid-row">
-        <div class="grid-col-12">
-
-          <section class="stream">
-
-            {{/* Future Events ===========================  */}}
-            {{- $events := where .Site.RegularPages.Reverse "Section" "events" -}}
+      <section class="stream" aria-label="Featured news and events">
```

Comment:
Changed this to a more semantic `<section>` and added an aria label describing it.